phone/src/os/phone/hooks/usePhone.ts

```diff
@@ -1,14 +1,14 @@
 import { useRecoilValue } from 'recoil';
 import { phoneState } from './state';
-import { ServerConfig } from '../../../common/typings/config';
+import { IServerConfig } from '../../../common/typings/config';
+import ServerConfig from '../../../config.autogen.json';
 
 interface IUsePhone {
   visibility: boolean;
-  config?: ServerConfig;
+  config?: IServerConfig;
 }
 
 export const usePhone = (): IUsePhone => {
   const visibility = useRecoilValue(phoneState.visibility);
-  const config = useRecoilValue(phoneState.phoneConfig);
-  return { visibility, config };
+  return { visibility, config: ServerConfig as any };
```

Comment:
This will assign `config` a type of IServerConfig rather than any as previous

```suggestion
  const config = ServerConfig as IServerConfig
  return { visibility, config };
```

Reply:
This has issues with enum and not string types because it's coming from json. That's why I left any first.

Reply:
Ah I see the use case then, if we are going to use Enum types should we just move it a .ts file?

Reply:
Let's merge and improve that in another iteration.